moztelemetry/dataset.py

```diff
@@ -231,7 +258,8 @@ def summaries(self, sc, limit=None):
         keys = sc.parallelize(scanned).flatMap(self.store.list_keys)
         return keys.take(limit) if limit else keys.collect()
 
-    def records(self, sc, limit=None, sample=1, seed=42, decode=None, summaries=None):
+    def records(self, sc, limit=None, sample=1, seed=42, decode=None,
+            summaries=None, group_by='greedy', 'equal_size'):
```

Comment:
There should only be value: `group_by='greedy'`. Please add to the doc-string to reflect the updated parameters.